__init__.py

```diff
@@ -253,22 +314,23 @@ def connect_ui(self, chat_control):
             log.debug(account_name + " ⇒ Adding OMEMO ui for " + contact_jid)
             omemo_enabled = state.encryption.is_active(contact_jid)
             self.ui_list[account_name][contact_jid] = Ui(self, chat_control,
-                                                         omemo_enabled)
+                                                         omemo_enabled, state)
         else:
             log.warn(account_name + " ⇒ No OMEMO dev_keys for " + contact_jid)
 
-    def are_keys_missing(self, contact):
+    def are_keys_missing(self, account_name, contact_jid):
         """ Used by the ui to set the state of the PreKeyButton. """
-        account = contact.account.name
-        my_jid = gajim.get_jid_from_account(account)
-        state = self.get_omemo_state(account)
+
+        my_jid = gajim.get_jid_from_account(account_name)
+        state = self.get_omemo_state(account_name)
         result = 0
-        result += len(state.devices_without_sessions(str(contact.jid)))
+        result += len(state.devices_without_sessions(str(contact_jid)))
         result += len(state.own_devices_without_sessions(my_jid))
         if result > 0:
-            log.warn(account + " ⇒ Missing keys for " + contact.jid + ": " +
+            log.warn(account_name + " ⇒ Missing keys for " + contact_jid + ": " +
                      str(result))
-        return result
+            log.warn('query keys now ...')
```

Comment:
I think we should change both of the logs to `info`. A missing key is not that exceptional and is part of the normal workflow now